webservices/common/models/itemized.py

```diff
@@ -145,7 +144,8 @@ class ScheduleA(BaseItemized):
     national_committee_nonfederal_account = db.Column('national_cmte_nonfed_acct', db.String)
 
     # Transaction meta info
-    election_type = db.Column('election_tp', db.String) # ? election_type looks like it's included in BaseItemized already
+    election_type = db.Column('election_tp', db.String)
+    # ? election_type looks like it's included in BaseItemized already
```

Comment:
Do you have time to resolve this comment?

Reply:
I didn't see `election_type` in BaseItemized. I remove the comment. Can you confirm it?Thanks @lbeaufort